### PR TITLE
add/remove orderer docs

### PR DESCRIPTION
**Pull request by semil**
Docs regarding the addition and removal of an orderer to/from existing cluster or channel.

#### Type of change

- Documentation update

#### Related issues

Issue: #4332

### CODE REVIEW

**Comment by denyeart** [comment]
This is a much needed addition. Before I do a detailed review, let me ask if you've seen the similar "Adding an Org to a Channel" tutorial - https://hyperledger-fabric.readthedocs.io/en/latest/channel_update_tutorial.html

"Adding an Org to a Channel" tutorial has been improved over time and has a good mix of conceptual explanation and tutorial instructions. It may make sense to use the same format and copy/paste some of the background information, and then extend/replace with your "adding a consenter" content. For example "Adding an Org to a Channel" tutorial assumes that the reader already has test network up and running, and then walks the reader through how to extend it, while your tutorial has them generate everything including for the 5th consenter and then bring up a new channel. Following the "Adding an Org to a Channel" tutorial approach may make more sense for an "adding" tutorial. Also it would be nice if the sections aligned to each other and similar wording was used. Although you may have deviated on purpose? Just wanted to get your initial thoughts on making these two tutorials similar in nature, or if you intentionally wanted it to be a little different.

File: docs/source/create_channel/add_orderer.md [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+    "consensusRelation": "follower",
+    "status": "onboarding",
+    "height": 0
+}
+```
+
+You should see something similar to the following in your orderer logs:
+```
+INFO [orderer.consensus.smartbft.chain] NewChain -> SmartBFT-v3 is now servicing chain mychannel channel=mychannel
+INFO [orderer.common.cluster] ConfigureNodeCerts -> Updating nodes identity, channel: mychannel, nodes: [id:1 host:"orderer.example.com" port:7050 msp_id:"OrdererMSP" identity:"..." client_tls_cert:"..." server_tls_cert:"..." ]
```
Comment (yacovm) [inline docs/source/create_channel/add_orderer.md]: the configure node certs thing is redundant, no need to put it here.

File: docs/source/create_channel/add_orderer.md [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+    "url": "/participation/v1/channels/mychannel",
+    "consensusRelation": "follower",
+    "status": "onboarding",
+    "height": 0
+}
+```
+
+You should see something similar to the following in your orderer logs:
+```
+INFO [orderer.consensus.smartbft.chain] NewChain -> SmartBFT-v3 is now servicing chain mychannel channel=mychannel
```
Comment (yacovm) [inline docs/source/create_channel/add_orderer.md]: apart from this, you can put a print of the follower (or if it's a leader) sending/receiving heartbeats from/to the other nodes

File: docs/source/create_channel/add_orderer.md [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+INFO [orderer.common.cluster] ConfigureNodeCerts -> Updating nodes identity, channel: mychannel, nodes: [id:1 host:"orderer.example.com" port:7050 msp_id:"OrdererMSP" identity:"..." client_tls_cert:"..." server_tls_cert:"..." ]
+```
+
+You can read further about the osnadmin command [here](https://hyperledger-fabric.readthedocs.io/en/latest/create_channel/create_channel_participation.html#step-two-use-the-osnadmin-cli-to-add-the-first-orderer-to-the-channel).
```
Comment (yacovm) [inline docs/source/create_channel/add_orderer.md:298]: You have a "# Adding Orderer To An Existing Network" title but there is no transition to removing an orderer. 

Can you add a "# Removing an orderer from an existing network"?

File: docs/source/create_channel/add_orderer.md [hunk 4]
```diff
@@ -0,0 +1,362 @@
...
+
+## Use the `osnadmin` CLI to remove the new orderer from the test channel
```
Comment (yacovm) [inline docs/source/create_channel/add_orderer.md]: You don't need this ## subtitle.

**Review by yacovm — APPROVED** [review]

**Comment by yacovm** [comment]
I merged this, @denyeart please take a look and if you want to make changes, feel free to make a new PR

**Comment by satota2** [comment]
Hello @yacovm @denyeart @semil

> I merged this, @denyeart please take a look and if you want to make changes, feel free to make a new PR

I'm been trying out the BFT samples in fabric-samples recently.
As part of this process, I also tried to add an orderer following the instructions in this document.

I didn't get a chance to comment on the recent pull request in time, but I've identified some areas from a user's perspective that might benefit from further improvements. I'll be submitting a new PR with these suggestions shortly.

**Pull request merged**